config/efr32/lib/pw_rpc/BUILD.gn

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+import("//build_overrides/pigweed.gni")
+import("$dir_pw_build/target_types.gni")
+
+static_library("pw_rpc") {
+  output_name = "libPwRpc"
+
+  public_configs = [ "${dir_pigweed}/pw_hdlc:default_config" ]
+  sources = [ "${dir_pigweed}/pw_hdlc/rpc_example/hdlc_rpc_server.cc" ]
+  deps = [
+    "$dir_pw_rpc:server",
+    "$dir_pw_rpc/nanopb:echo_service",
+    "$dir_pw_rpc/system_server",
+    "${chip_root}/examples/platform/efr32/pw_sys_io:pw_sys_io_efr32",
+    "${dir_pigweed}/pw_hdlc:pw_rpc",
+    dir_pw_assert,
+    dir_pw_hdlc,
+    dir_pw_log,
+  ]
+
+  output_dir = "${root_out_dir}/lib"
+
+  complete_static_lib = true
```

Comment:
Why complete_static_lib ?

Reply:
No use, dragged this from a copy of a implemented static_library("pw_rpc") . I will delete it.